Pyramide slide down.py

```diff
@@ -0,0 +1,8 @@
+pyramid = [[3], [7, 4], [2, 4, 6], [8, 5, 9, 3]]
+for i in range(len(pyramid)-2, -1, -1):
+    for j in range(i+1):
+        pyramid[i][j] += max(pyramid[i+1][j], pyramid[i+1][j+1])
+print(pyramid[0][0])
```

Comment:
This solution definitely works, at the same time, it's complicated to read and follow. I suggest making it more readable by using slicing, extra variables and 

```suggestion
pyramid = [[3], [7, 4], [2, 4, 6], [8, 5, 9, 3]]
reversed_pyramid = pyramid[::-1]

for current_level, next_level in zip(reversed_pyramid[:-1], reversed_pyramid[1:]):
    for i in range(len(current_level) - 1):
        left, right = current_level[i], current_level[i+1]
        next_level[i] += max(left, right)

print(pyramid[0][0])
```